framer/Utils.coffee

```diff
@@ -353,6 +353,11 @@ Utils.isTouch = ->
 	window.ontouchmove is null and
 	window.ontouchend is null
 
+Utils.isMouse = ->
+	window.onmousedown is null and
+	window.onmouseup is null and
+	window.onmousemove is null
+
```

Comment:
Does this work? I’d expect for backwards compatibility this would still return `true` on iOS?